src/wallet/init.cpp

```diff
@@ -194,7 +194,10 @@ bool VerifyWallets()
     }
 
     if (gArgs.IsArgSet("-walletdir") && !fs::is_directory(GetWalletDir())) {
-        return InitError(strprintf(_("Error: Specified wallet directory \"%s\" does not exist."), gArgs.GetArg("-walletdir", "").c_str()));
+        if (fs::exists(GetWalletDir())) {
+            return InitError(strprintf(_("Specified wallet directory \"%s\" is not a directory."), gArgs.GetArg("-walletdir", "").c_str()));
```

Comment:
IMO this is better `Specified -walletdir \"%s\" is not a directory` (no period and references parameter name).

Reply:
Done